dist/images/ovnkube.sh

```diff
@@ -1203,6 +1205,12 @@ ovn-master() {
   fi
   echo "multi_network_enabled_flag=${multi_network_enabled_flag}"
 
+  network_segmentation_enabled_flag=
+  if [[ ${ovn_network_segmentation_enable} == "true" ]]; then
+	  network_segmentation_enabled_flag="--enable-multi-network --enable-network-segmentation"
+  fi
+  echo "network_segmentation_enabled_flag=${network_segmentation_enabled_flag}"
```

Comment:
We have to also add ${network_segmentation_enabled_flag}  to `  /usr/bin/ovnkube --init-ovnkube-controller`  and friends.

Reply:
good catch quique!